src/lib/i18n/translate.ts

```diff
@@ -86,3 +114,31 @@ export type TResolvable = SkyraArgs | TFunction;
 export function resolveT(t: TResolvable): TFunction {
 	return typeof t === 'function' ? t : t.t;
 }
+
+/**
+ * Returns a translation function for the specified locale, or the default 'en-US' if none is provided.
+ * @param locale The locale to get the translation function for.
+ * @returns The translation function for the specified locale.
+ */
+export function getT(locale?: LocaleString | Nullish) {
```

Comment:
Same as previous comment, this function has stricter types, requiring `LocaleString` rather than just any `string`. Also provides a default so `getT()` is valid and gets the default language.